geomstats/geometry/special_euclidean.py

```diff
@@ -765,7 +765,6 @@ def exponential_matrix(self, rot_vec):
             term_2[i] = gs.matmul(skew_rot_vec[i], skew_rot_vec[i]) * coef_2[i]
```

Comment:
Good! By the way this method is not unit tested. It is a bit surprising to find it here though, shouldn't it be in SpecialOrthogonal?

Reply:
It is in this file because it is an auxiliary function for the exponential barycenter of SE(n). I have converted it to a private method, to make this more clear. If it gets used elsewhere, I agree that we could think about moving it.